web/modules/custom/joinup_core/src/JoinupRelationManager.php

```diff
@@ -123,4 +134,122 @@ public function getParentElibrary(EntityInterface $entity) {
     return $e_library;
   }
 
+  /**
+   * Retrieves all the users that have the administrator role in a group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The group entity.
+   * @param array $state
+   *   (optional) An array of membership states to retrieve. Defaults to active.
+   *
+   * @return array
+   *   An array of users that are administrators of the entity group.
+   */
+  public function getGroupOwners(EntityInterface $entity, array $state = [OgMembershipInterface::STATE_ACTIVE]) {
+    $role_id = $entity->getEntityTypeId() . '-' . $entity->bundle() . '-administrator';
+
+    $users = [];
+    foreach ($this->getGroupMemberships($entity, $state) as $membership) {
+      $user = $membership->getUser();
+      if (!empty($user) && $membership->hasRole($role_id)) {
+        $users[$user->id()] = $user;
+      }
+    }
+
+    return array_filter($users);
```

Comment:
Ahh sorry! I was looking to the next method. Indeed this array_filter is not needed. I noticed that as well at some point but the ticket was already in QA.
I will remove it to make sure that no tests are failing.